app/views/users/index.html.erb

```diff
@@ -2,6 +2,6 @@
 <h1>All accounts</h1>
 <%= will_paginate %>
 <ul class="users">
-  <%= render @users %>
+  <%= render @users.paginate(page: params[:page], per_page: 8) %>
```

Comment:
This looks ok. You can also specify the number of records per page in the `User` model instead.
```ruby
self.per_page = 8
```